snap/snapcraft.yaml

```diff
@@ -66,6 +67,8 @@ apps:
         - dbus-freedesktop-impl-portal-gtk
 
 plugs:
+  audio-playback: null
+  audio-record: null
```

Comment:
These should be slots, since we'd expect e.g. `firefox:audio-playback` to connect to `ubuntu-desktop-session:audio-playback`. We should also have a pulseaudio slot to cover old snaps that haven't udpated to audio-playback/audio-record.

Reply:
Again, that would be when finally we get pipewire working inside the desktop container.